wolfssl/wolfio.h

```diff
@@ -163,11 +163,15 @@
             #include <sys/socket.h>
             #include <arpa/inet.h>
             #include <netinet/in.h>
-            #include <netdb.h>
+            #ifdef HAVE_NETDB_H
+                #include <netdb.h>
+            #endif
             #ifdef __PPU
                 #include <netex/errno.h>
             #else
-                #include <sys/ioctl.h>
+                #ifdef HAVE_SYS_IOCTL_H
```

Comment:
@dgarske @douzzer 
Apparently, this check is removed in the current master.
Could you add it back please in `src/wolfio.c:85` ?
I can open a PR, but it might take forever to get legal approval.
Thanks!

Reply:
Hi @farazrbx , I don't recall that getting changed back, but we will take a look. Thanks, David Garske, wolfSSL

Reply:
https://github.com/farazrbx/wolfssl/commit/897a8419c10fbe7ea3d0ce32fbcb6a9b3b9c367f

Reply:
https://github.com/wolfSSL/wolfssl/pull/7288/files